prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -706,6 +732,8 @@ class Hl7Serializer(
         rawFacilityName: String,
     ) {
         terser.set(formPathSpec("ORC-21-1"), rawFacilityName.trim().take(50))
+        // setting a default value for ORC-21-2 per PA's request.
+        terser.set(formPathSpec("ORC-21-2"), DEFAULT_ORGANIZATION_NAME_TYPE_CODE)
```

Comment:
Is there going to be a problem for anyone else with this populated where it wasn't before?

Reply:
@clediggins-usds typically filling in a value like this isn’t going to cause any issues. This is just additional metadata.